youtube_dl/extractor/sbs.py

```diff
@@ -44,12 +44,13 @@ class SBSIE(InfoExtractor):
             (["\'])(?P<url>https?://(?:www\.)?sbs\.com\.au/ondemand/video/.+?)\1''']
 
     _TESTS = [{
-        # Original URL is handled by the generic IE which finds the iframe:
+        # Exceptional unrestricted show for testing, thanks SBS,
+        # from an iframe of this page, handled by the generic IE, now 404:
         # http://www.sbs.com.au/thefeed/blog/2014/08/21/dingo-conservation
         'url': 'http://www.sbs.com.au/ondemand/video/single/320403011771/?source=drupal&vertical=thefeed',
         'md5': 'e49d0290cb4f40d893b8dfe760dce6b0',
         'info_dict': {
-            'id': '320403011771',  # '_rFBPRPO4pMR',
+            'id': '320403011771',  # formerly '_rFBPRPO4pMR', no longer found
```

Comment:
That url still shows this id on yt-dlp  (and downloads correctly). I will test tomorrow to see if it's a potential regression with this pr

Reply:
As it's an old show, it's still available via ThePlatform, but its ID there doesn't seem to have survived being imported into the DB(s) underlying the new media APIs.